helm/polaris/README.md

```diff
@@ -0,0 +1,119 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+<!---
+  This README.md file was generated with:
+  https://github.com/norwoodj/helm-docs
+  Do not modify the README.md file directly, please modify README.md.gotmpl instead.
+  To re-generate the README.md file, install helm-docs then run from the repo root:
+  helm-docs --chart-search-root=helm
+-->
+
+# Polaris Helm chart
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
+
+A Helm chart for Polaris.
+
+**Homepage:** <https://polaris.io/>
+
+## Maintainers
+
+## Source Code
+
+* <https://github.com/apache/polaris>
+
+## Installation
+
+### Optional
+
+When using custom `persistence.xml`, you will need to create a K8S Secret for `.persistenceConfigSecret`. Here is a sample instruction:
```

Comment:
Don't use "you" in documentation. 

"Custom `persistence.xml` requires a Kubernetes Secret for `.pers...` 

Example Secret Creation:
```bash
...
```

Reply:
Fixed